roles/redhat-subscriptions/README.md

```diff
@@ -0,0 +1,45 @@
+Ansible Role for Registering and Subscribing content
+====================================================
+
+This Ansible role registers and subscribes Satellite6 node with RHN.
+
+Requirements
+------------
+
+No Requirements are required for this role.
+
+Role Variables
+--------------
+
+Requires rhsm_username and rhsm_password to be set in vars/satellite_common.yml if using internal_repo or CDN.
+
+Requires dogfood_ak and dogfood_org to be set in vars/satellite_63.yml if using internal_ak.
```

Comment:
I would not include the specific mention that it requires `vars/satellite_63.yml` and the above `satellite_common` because these are roles and they don't care what vars file you use.